databases/week2/wk2_part3.sql

```diff
@@ -0,0 +1,37 @@
+--Getting all the tasks assigned to users whose email ends in @spotify.com
+SELECT name AS user, title AS task
+FROM user_task
+INNER JOIN task
+ON user_task.task_id= task.id
+INNER JOIN user
+ON user_task.user_id= user.id
+WHERE email LIKE '%@spotify.com';
+--4 rows returned.
+
+--Getting all the tasks for 'Donald Duck' with status 'Not started'
+SELECT user.name AS user, title AS task,status.name AS status
+FROM user_task
+INNER JOIN task
+ON user_task.task_id= task.id
+INNER JOIN status
+ON task.status_id = status.id
+INNER JOIN user
+ON user_task.user_id= user.id
+WHERE user.name = 'Donald Duck' AND status.name = 'Not started';
+--2 rows returned.
+
+--Getting all the tasks for 'Maryrose Meadows' that were created in september.
+SELECT name AS user, title AS task
+FROM user_task
+INNER JOIN task
+ON user_task.task_id= task.id
+INNER JOIN user
+ON user_task.user_id= user.id
+WHERE user.name = 'Maryrose Meadows' AND month(created)=09;
```

Comment:
I don't have a user_task table, so I can't test your query. you can write with only one join.